src/core/waf.cc

```diff
@@ -46,6 +46,25 @@
 #include <unistd.h>
 #include <dirent.h>
 #include <errno.h>
+
+namespace {
```

Comment:
Why the anonymous namespace here?  Could we just move this into the `ns_waflz` namespace?